source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/patterns/patterns_cfg.py

```diff
@@ -70,7 +70,12 @@ class GridPatternCfg(PatternBaseCfg):
 
 @configclass
 class PinholeCameraPatternCfg(PatternBaseCfg):
-    """Configuration for a pinhole camera depth image pattern for ray-casting."""
+    """Configuration for a pinhole camera depth image pattern for ray-casting.
+
+    ..note ::
```

Comment:
Might be better to link it to the Omniverse docs page so people know where all this is coming from. We SHOULD make this a warn or caution notice so that users know that even if the doc says `mm` in USD side, it is actually a-tenth of a cm stage unit.

Might be more informative than right now.